subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -328,17 +336,42 @@ static void handle_wifi_ap_disable_result(struct net_mgmt_event_callback *cb)
 	} else {
 		print(context.sh, SHELL_NORMAL, "AP disabled\n");
 	}
+
+	k_mutex_lock(&wifi_ap_sta_list_lock, K_FOREVER);
+	for (int i = 0; i < CONFIG_WIFI_SHELL_MAX_AP_STA; i++) {
+		if (!sta_list[i].valid) {
+			continue;
+		}
+
+		sta_list[i].valid = false;
+	}
```

Comment:
Could we just do `memset(&sta_list, 0, sizeof(sta_list));` here as the end result will be the same?